uma/protocol.py

```diff
@@ -434,3 +434,28 @@ def _from_dict(cls, json_dict: Dict[str, Any]) -> Dict[str, Any]:
 class UtxoWithAmount(JSONable):
     utxo: str
     amount_msats: int
+
+
+@dataclass
+class PostTransactionCallback(JSONable):
+    utxos: List[UtxoWithAmount]
+    """List of utxo/amounts corresponding to the VASPs channels."""
+
+    vasp_domain: str
+    """
+    The domain of the VASP that is sending the callback. Used by the VASP to fetch the public keys of
+    its counterparty.
+    """
+
+    signature: str
+    """Signature is the base64-encoded signature of sha256(Nonce|Timestamp)"""
+
+    signature_nonce: str
+    """Random string that is used to prevent replay attacks"""
+
+    signature_timestamp: int
```

Comment:
datetime is not fun to serialize so keeping this int is probably the way to go

Reply:
Fair enough for now!